data/formats-data.js

```diff
@@ -2183,6 +2183,16 @@ exports.BattleFormatsData = {
 		],
 		tier: "New",
 	},
+	smearglepokestar: {
```

Comment:
I really don't think it should be considered a Smeargle forme just because it's inaccessible.

I also don't like the idea of its learnsets being categorized as event learnsets, since "event" means "accessible in normal gameplay" to me, but I'll allow it as long as it's categorized as a separate species.